src/uct/api/v2/uct_v2.h

```diff
@@ -281,7 +281,14 @@ typedef enum {
      * in order for @ref UCT_MD_MEM_DEREG_FLAG_INVALIDATE flag to function
      * the @ref UCT_MD_MKEY_PACK_FLAG_INVALIDATE flag must be set.
      */
-    UCT_MD_MKEY_PACK_FLAG_INVALIDATE = UCS_BIT(0)
+    UCT_MD_MKEY_PACK_FLAG_INVALIDATE = UCS_BIT(0),
+
+    /**
+     * The flag is used to indicate that the memory region should be packed to
+     * be accessed by another process using the same device to perform UCT
```

Comment:
should be packed in order to be accessed ... ?

Reply:
done